lib/clock.dart

```diff
@@ -12,35 +12,33 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-part of quiver.time;
+import 'src/utils.dart';
 
 /// Returns current time.
+@Deprecated("Pass around an instance of Clock instead.")
 typedef DateTime TimeFunction();
 
 /// Return current system time.
+@Deprecated("Use new DateTime.now() instead.")
 DateTime systemTime() => new DateTime.now();
 
-/// Provides points in time relative to the current point in time, for example:
-/// now, 2 days ago, 4 weeks from now, etc.
+/// A class which encapsulates the notion of the "current time" and provides
```

Comment:
Calling it "the current time" makes me think that it should *implement* `DateTime` rather than *providing* it. What about "A provider for the 'current time'..."?

Reply:
"A provider for" SGTM